executor/show.go

```diff
@@ -166,11 +166,28 @@ func (e *ShowExec) fetchShowEngines() error {
 	return nil
 }
 
+// move information_schema to the first, and the other are sorted in the origin ascending order.
+func moveInfoSchemaToFront(dbs []string) {
+	if strings.EqualFold(dbs[0], "INFORMATION_SCHEMA") {
+		return
+	}
+
+	for i := 1; i < len(dbs); i++ {
```

Comment:
Could you try binary search `SearchStrings` here? It won't increase the coding complexity.

Reply:
Thanks, could have a try.